src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigForm.tsx

```diff
@@ -107,20 +123,24 @@ export const AutomatedAnalysisConfigForm: React.FC<AutomatedAnalysisConfigFormPr
   const [maxSize, setMaxSize] = React.useState(recordingConfig.maxSize);
```

Comment:
Can we wrap these form states into a single object?

Like

```tsx
interface FormConfig {
  maxAge: number;
  maxAgeUnit: number; 
  maxSize: number;
  maxSizeUnit: number; 
  template: Pick<Partial<EventTemplate>, 'name' | 'type'>
}
```

Reply:
Probably this needs to be done other views too but for 2.3 now, just this view is needed.